src/Bootstrap/BootstrapAbstract.php

```diff
@@ -152,6 +195,9 @@ protected function initEnvironment()
     {
         error_reporting(-1);
         date_default_timezone_set('UTC');
+        // TODO: Move these to the docker images [emul]
+        ini_set('xdebug.overload_var_dump', 0);
```

Comment:
Definitely we need to move these out of here.

I'll try to find a way to set it via some env variables or something like that, as this is primarily a developer preference setting. I for example prefer to have xdebug overload the var_dump.